### ISSUE TITLE
bootloader: Add "Wipe device" button

### ISSUE THREAD

**Issue by slush0** [trezor/trezor-firmware#60]
Allow to wipe device without connecting to the computer.

Related to #301, where USB is disabled in bootloader, and complement to on-device management #302 in cases PIN is not known.

**Comment by tsusanka** [comment]
If we decide to do #1039 this is not needed.

**Comment by tsusanka** [comment]
Product wants to do this. I am removing also "UX needed", because if we wish to do some more UX/design modifications we should go with #1049.

**Comment by tsusanka** [comment]
Let's discuss where exactly the Wipe Button should be. 

![Screenshot from 2020-06-05 15-34-08](https://user-images.githubusercontent.com/1835345/83882186-0a420380-a742-11ea-9ed4-569d982ea46a.png)

**Comment by tsusanka** [comment]
Product has decided we want to do this in #1049.

**Comment by Hannsek** [comment]
Until now, if you stole the trezor, you couldn't use it. With this, you can wipe the stolen trezor and start using it. Am I right?

**Comment by prusnak** [comment]
> Until now, if you stole the trezor, you couldn't use it. With this, you can wipe the stolen trezor and start using it. Am I right?

No. You can still run `trezorctl device wipe` when the device is in the bootloader mode to wipe the device without knowing the PIN.

**Comment by Hannsek** [comment]
ok, thanks. will be solved by this: 
<img width="391" alt="image" src="https://user-images.githubusercontent.com/57008159/219069659-fa565c36-18c4-4dd7-9b5f-35142f9669a6.png">
